### DISCUSSION TITLE
no presist documentation

### DISCUSSION THREAD

**Discussion by amirreza-sf80** [valkey-io/valkey#1134] [Q&A] [answered]
hi
please add documentation on how to configure valkey as not persistent (no RDB and no AOF)

**Comment by madolson** [comment]
https://valkey.io/topics/persistence/ Exists. Does it help explain?

**Reply by amirreza-sf80** [reply]
hi
i honestly don't see any explanation about no-persistent mode
if I'm being blind i apologize in advance

**Reply by madolson** [reply, marked as answer]
I'm sorry. I read your comment and thought you were saying that there was no documentation about persistence. Your actual comment was about documentation for when persistence is disabled. My apologies. I don't think have any documentation at the moment. 

Basically just start up the server with `--save ""` and persistence will be disabled.

**Reply by melroy89** [reply]
Or add `save ""` to the config file (and remove all existing `save` lines being present).

**Reply by melroy89** [reply]
PR created: https://github.com/valkey-io/valkey-doc/pull/179

**Reply by zuiderkwast** [reply]
I have wished many times that `save ""` was the default. Especially when testing small things locally.

It's somewhat unintuitive to pass an empty string. It is what it is though. Documenting it explicitly is a great idea.

**Reply by ranshid** [reply]
maybe we can add a --noautosave argument to help those of us who hates it?

**Reply by zuiderkwast** [reply]
Yes, but is --noautosave easier to type than `save ''`? The main problem is that you forget to type it. If there is a way to set config by environement variables, then I can add `save ""` to my `.bashrc`.

**Reply by melroy89** [reply]
The person who created this discussion just didn't know how-to do it in the first place. Hence [my PR](https://github.com/valkey-io/valkey-doc/pull/179) for just documenting the feature. I also don't think `--noautosave` is much easier.

**Reply by zuiderkwast** [reply]
Thinking about this again, I realize you can use (diskless) replication and rely on this instead to make sure the data is safe, so the use case is not only for caching. Maybe we should have mentioned this too... Or do people figure this out by themselves?

**Reply by melroy89** [reply]
Well.. I does say "for example" now. I wanted to keep the "No persistence" section as brief as possible. To keep to the point, which is important due to all the social media, people the attention curve.. hello, you still reading this?? (just joking now)

Despite that, you do have a valid use-case there.. what about a different section or better.. a **different page** in the documentation? There might be more advance configurations & setups you might want to share?

**Reply by zuiderkwast** [reply]
Yeah, brief is good. We have the replication page for example.
